clusterloader2/pkg/prometheus/manifests/exporters/node_exporter/nodeExporter-serviceMonitor.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app.kubernetes.io/component: exporter
+    app.kubernetes.io/name: node-exporter
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 1.3.1
+  name: node-exporter
+  namespace: monitoring
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 15s
```

Comment:
What was maximum node scale you tested with interval: 15s?

I suspect that prometheus may have a problem with scraping e.g. 15k nodes with 15s interval with a single instance.

Reply:
I guess we may need some tricky logic like in 

https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/prometheus/manifests/default/prometheus-serviceMonitorKubeProxy.yaml#L18

if we expect increasing interval.